.github/workflows/bootstrapping_ci.yml

```diff
@@ -14,18 +14,18 @@ on:
 
 concurrency:
   group: ${{ github.workflow }}-${{ github.ref == 'refs/heads/master' && github.sha || github.ref }}
-  cancel-in-progress: false
+  cancel-in-progress: true
 
 jobs:
   bootstrap-v:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-14]
+      fail-fast: false
```

Comment:
From <https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#handling-failures>:
> jobs.<job_id>.strategy.fail-fast applies to the entire matrix. If jobs.<job_id>.strategy.fail-fast is set to true or its expression evaluates to true, GitHub will cancel all in-progress and queued jobs in the matrix if any job in the matrix fails. This property defaults to true.

What happens when the jobs themselves do not fail, but they are in the same concurrency group, and are started at the same time?

See <https://github.com/vlang/v/actions/runs/8890038625/attempts/1> .
Both jobs got canceled.

Reply:
Okay I'm happy to be able to address all the concerns. Gonna start here:

- The `fail-fast: false` is only if mac fails it won't cancel the ubuntu matrix, its unrelated to the concurrency group. To get the result for both platforms to better analyze why a failed happened if it did so. `fail-fast: false` is what we use afaik in any other matrix.